socorro/unittest/external/es/base.py

```diff
@@ -963,7 +969,7 @@ def index_many_crashes(
         )
         self.refresh_index()
 
-    def refresh_index(self):
+    def refresh_index(self, index=None):
```

Comment:
That variable is usually called ``es_index`` throughout the app, I think it would be beneficial to keep that convention.